### PR TITLE
Updated projects page

### PR DESCRIPTION
**Pull request by keith-turner**

### CODE REVIEW

**Comment by keith-turner** [comment]
@rfecher @joshelser @dlmarion @meiercaleb @elahrvivaz let me know if you have any issues with the descriptiong of the projects.

**Comment by dlmarion** [comment]
LGTM

**Comment by joshelser** [comment]
There's also [Apache Pig integration](http://pig.apache.org/docs/r0.16.0/func.html#AccumuloStorage)

File: projects.md [hunk 1]
```diff
@@ -8,6 +8,49 @@ by other users within the community. As such, we're happy to curate a list of pr
...
+[Gora](https://gora.apache.org/) open source framework provides an in-memory data model and persistence for big data.  Accumulo's continuous ingest test suite was adapted to Gora and called [Goraci](http://gora.apache.org/current/index.html#goraci-integration-testsing-suite).
+
+#### Graphulo
+
+Graphulo is a Java library for Apache Accumulo which delivers server-side sparse matrix math primitives that
+enable higher-level graph algorithms and analytics. [Code](https://github.com/Accla/graphulo).
+
+#### Hive
+
+[Hive](https://hive.apache.org/) data warehouse software facilitates reading, writing, and managing large datasets residing in distributed storage using SQL.  Hive has [handlers for Accumulo](https://cwiki.apache.org/confluence/display/Hive/AccumuloIntegration).
```
Comment (joshelser) [inline projects.md]: Suggest: "Hive has the ability to read and write data in Accumulo using the [AccumuloStorageHandler](https://cwiki.apache.org/confluence/display/Hive/AccumuloIntegration)"

**Comment by rfecher** [comment]
description looks good, thanks @keith-turner

**Pull request merged**